packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -28,6 +28,16 @@ interface ISnapshotSection {
     sequenceNumber: number;
 }
 
+/**
+ * The parsing is significantly faster if the position of props is well known instead of dynamic. So
+ * fast variables represent how many times faster parsing path is executed and slow variables represents
+ * how many times slower path is executed.
+ */
+let fastBlobPath = 0;
```

Comment:
globals are bad. You will get wrong result if we parse snapshots many times in a session, which is very likely to happen (counting summarizer).

Reply:
I now added it to trees latest telemetry.